apps/qubit/modules/taxonomy/actions/indexAction.class.php

```diff
@@ -94,7 +96,12 @@ public function execute($request)
       }
 
       // Show alert
-      $message = $this->context->i18n->__("We've redirected you to the first page of results. To avoid using vast amounts of memory, AtoM limits pagination to 10,000 records. To view the last records in the current result set, try changing the sort direction.");
+      $message = $this->context->i18n->__(
+        "We've redirected you to the first page of results.
+        To avoid using vast amounts of memory, AtoM limits pagination to %1% records.
+        To view the last records in the current result set, try changing the sort direction.",
+        array('%1%' => $maxResultWindow)
+      );
       $this->getUser()->setFlash('notice', $message);
 
       // Redirect to fist page
```

Comment:
Wow, there's a lot of repetition of this code.  Can it be refactored into a base class?

Reply:
I agree, but there are slight differences in the messages and I don't want to go much further in this PR.